pkg/labels/labels_test.go

```diff
@@ -19,6 +19,144 @@ import (
 	"github.com/prometheus/prometheus/util/testutil"
 )
 
+func TestLabels_String(t *testing.T) {
+	cases := []struct {
+		name     string
+		lables   Labels
+		expected string
+	}{
+		{
+			name: "normal",
+			lables: Labels{
+				{
+					Name:  "t1",
+					Value: "t1",
+				},
+				{
+					Name:  "t2",
+					Value: "t2",
+				},
+			},
+			expected: "{t1=\"t1\", t2=\"t2\"}",
+		},
+		{
+			name:     "empty lables",
+			lables:   Labels{},
+			expected: "{}",
+		},
+		{
+			name:     "nil lables",
+			lables:   nil,
+			expected: "{}",
+		},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			str := c.lables.String()
+			if str != c.expected {
+				t.Errorf("expected: %v, got: %v", c.expected, str)
+			}
+		})
+	}
+}
+
+func TestLabels_MarshalJSON(t *testing.T) {
```

Comment:
The tests for marshal and unmarshal of json would belong to the JSON library, it is not needed here.

Reply:
Thanks, i will delete it.